client/Android/Studio/freeRDPCore/src/main/cpp/android_cliprdr.c

```diff
@@ -252,26 +253,28 @@ static UINT android_cliprdr_server_format_list(CliprdrClientContext* cliprdr,
 		}
 	}
 
+	bool hasUnicodeText = false;
```

Comment:
windows always has `CF_UNICODETEXT`, `CF_TEXT` (or `CF_OEM_TEXT`) are not utf-8 but some local encoding, so better ignore these

Reply:
and yes, `CF_UNICODE` is always UTF16-le